rosdep/base.yaml

```diff
@@ -781,6 +781,8 @@ gazebo:
     xenial: [gazebo7]
     yakkety: [gazebo7]
     zesty: [gazebo7]
+    artful: [gazebo9]
+    bionic: [gazebo9]
```

Comment:
CI is failing because these are not in alphabetical order

Reply:
Thanks Mika. Fixed.